matchms/filtering/reduce_to_number_of_peaks.py

```diff
@@ -0,0 +1,53 @@
+from math import ceil
+from ..Spikes import Spikes
+from ..typing import SpectrumType
+
+
+def reduce_to_number_of_peaks(spectrum_in: SpectrumType, n_required=1, n_max=100,
+                              ratio_desired=None) -> SpectrumType:
+    """Lowest intensity peaks will be removed when it has more peaks than desired.
+
+    Args:
+    ----
+    spectrum_in: matchms.Spectrum()
+        Input spectrum.
+    n_required: int
+        Number of minimum required peaks. Spectra with fewer peaks will be set
+        to 'None'. Default is 1.
+    n_max: int
+        Maximum number of peaks. Remove peaks if more peaks are found.
+    ratio_desired: float, optional
+        Set desired ratio between maximum number of peaks and parent mass.
+        Default is None.
+    """
+    def _set_maximum_number_of_peaks_to_keep(spectrum, ratio_desired):
```

Comment:
the pythonic way is to not have these arguments, just have 
```python
def _set_maximum_number_of_peaks_to_keep():
```

Reply:
Done.